### DISCUSSION TITLE
No auth.lck or authcode.txt

### DISCUSSION THREAD

**Discussion by JelenaRaskovska** [Pryaxis/TShock#2586] [Q&A]
Heya, I don't know if I've missed something or if I'm just stupid, but my TShock directory doesn't have an auth.lck file or authcode.txt, and my auth code doesn't appear in the command prompt for the server. Any pointer would be helpful, thank you.

**Comment by hakusaro** [comment]
Does it say that you’re running TShock when you start the server? Is it possible you didn’t get TShock into the plugins folder?
